src/partials/advertisement.html

```diff
@@ -1,2 +1,7 @@
 <section class="advertisement">
```

Comment:
please add class section to a class advertisement:
<section class="advertisement section">